Commands.md

```diff
@@ -361,6 +362,17 @@ Does the following:
   - Executes _.git/hooks/post-release.sh_ (if present)
 
 
+## git rename-branch
+
+Rename a branch (locally and remotely).
```

Comment:
@spacewander Technically after renaming the local, the branch is pushed to remote. So maybe the documentation can be a bit more clear on that front. Will make the update.

- [ ] update docs clarifying `rename` functionality